.yarnrc.yml

```diff
@@ -0,0 +1 @@
+yarnPath: ".yarn/releases/yarn-berry.js"
```

Comment:
https://yarnpkg.com/getting-started/install:
> Using a single package manager across your system has always been a problem. To be stable, installs need to be run with the same package manager version across environments, otherwise there's a risk we introduce accidental breaking changes between versions - after all, that's why the concept of lockfile was introduced in the first place! And with Yarn being in a sense your very first project dependency, it should make sense to "lock it" as well.
>
> For this reason, Yarn 2 and later are meant to be managed on a by-project basis. Don't worry, little will change!